.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows]
-        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, jruby]
+        ruby: [2.4, 2.5, 2.6, 2.7, jruby]
```

Comment:
Ruby 2.4 was eol in April, can we kill that too?
https://www.ruby-lang.org/en/downloads/branches/